### ISSUE TITLE
Add loaders with text on the loading screens

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/portal#363]
## Context
Currently when some screens on the portal load, there is no feedback to the user about what is happening on the portal screen. In most cases the data is to be fetched. Hence loaders should be added on these screens

### Requirement
* Add loader along with loading text to following screens and only display the content when its ready
    * Dashboard
    * Staking
    * Spacewalk - Bridge and Transfer screen
    * Nabla all screens (only Foucoco)
* The loader icon and text should be added to all screens for Pendulum, Amplitude and Foucoco
* Loader and text to be used of biggest font and size given in the linked wireframe

## Wireframe
https://www.figma.com/proto/5up5sghcttBogehUq4255i/App?type=design&node-id=4268-823&t=94mF8UWft6bDKobh-0&scaling=min-zoom&page-id=608%3A696

**Comment by prayagd** [comment]
@ebma @Sharqiewicz Can we show a single loading text and icon for the entire feature screen instead of showing smaller ones here and there
For ex. on staking feature i see there are three components, in this case we show the icon and text until all three components completely load. Is this possible?

**Comment by ebma** [comment]
Wouldn't we ideally use skeleton loaders for masking our content while it's loading? Example [here](https://skeletonreact.com/) to make clear what I'm referring to.

**Comment by prayagd** [comment]
@ebma yes you are right we do but its very unclear from user point of view whether something is happening on the screen or not as the there is not textual feedback 
Also some components load partially and show undefined as values until they are fetched for example see image below
![image](https://github.com/pendulum-chain/portal/assets/52105313/905bfcb9-6d6d-40b5-8f81-96a34a6e1c61)

**Comment by ebma** [comment]
But wouldn't these 'undefined' values be a perfect case for the skeleton loader? Maybe it's not used extensively enough and we need to make sure that it's used every time a value is still loading.

**Comment by prayagd** [comment]
@ebma agree but i am suggesting a more visual approach that is a loader icon with text, would that be too much?

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/portal/363?planning-poker) with Zenhub @ebma @Sharqiewicz
